libs/llm/llm.provider.service.ts

```diff
@@ -210,7 +220,27 @@ export class LLMProviderService implements OnModuleInit {
           availableModels,
         });
         break;
-
+      case 'gemini':
+        provider = new GeminiChatProvider({
+          provider: config.provider,
+          // baseURL: config.baseURL || this.config.get('GEMINI_BASEURL'),
+          model,
+          apiKey: config.apiKey || this.config.get('GEMINI_API_KEY'),
+          availableModels,
+        });
+        break;
+      case 'huggingface':
+        provider = new HuggingfaceChatProvider({
+          provider: config.provider,
+          baseURL: config.baseURL || this.config.get('HUGGINGFACE_BASEURL'),
```

Comment:
@lc-spxl credo che questo baseURL possa essere hard-coded

Reply:
If I got it right, HF supports custom endpoints pointing to dediated HW https://huggingface.co/inference-endpoints/dedicated